src/encoded/__init__.py

```diff
@@ -32,13 +32,29 @@
     json_asset
 )
 from dcicutils.log_utils import set_logging
+from dcicutils.beanstalk_utils import whodaman
+from encoded.commands.create_mapping_on_deploy import (
+    ENV_WEBPROD,
+    ENV_WEBPROD2,
+    BEANSTALK_PROD_ENVS,
+)
 import structlog
 import logging
 
 # location of environment variables on elasticbeanstalk
 BEANSTALK_ENV_PATH = "/opt/python/current/env"
 
 
+def get_mirror_env(settings):
```

Comment:
I feel like there are lots of odd assumptions built into this, ending up feeling weirdly ad hoc, but at least it's a small function and the logic is plain. Maybe a little more documentation and/or just comments in the code explaining the presumptions would fix that. But it's not urgent.